tests/Unit/Elliptic/Actions/Test_InitializeSystem.cpp

```diff
@@ -264,11 +184,6 @@ SPECTRE_TEST_CASE("Unit.Elliptic.Actions.InitializeSystem",
           // This check is against the source computed by the
           // analytic solution above
           get<0>(inertial_coords) + get<1>(inertial_coords));
-    CHECK(get(get_tag(LinearSolver::Tags::Operand<ScalarFieldTag>{})).size() ==
```

Comment:
There are several tests that the initialization has occurred as anticipated that have been removed now that the `operand_tag` and `operator_applied_to_operand_tag` etc are added to `InitializeElement.hpp` instead, but I didn't manage to find where these checks end up moved to. Is there a reason that you no longer need to require these quantities are initialized, or are they checked some other way by the tests of `InitializeElement.hpp`?

Reply:
The `InitializeElement` actions wasn't tested before because it contributed to a reduction at some point and I must have overlooked adding the test when I moved the reduction to `PrepareSolve`. Pushed another commit that adds the test